packages/devtools_app/lib/src/service/service_manager.dart

```diff
@@ -496,6 +498,36 @@ class ServiceConnectionManager {
   }
 }
 
+class ResolvedUriManager {
+  final _resolvedUrlMap = <String, String?>{};
+  final _unknownUris = <String>[];
+  VmServiceWrapper? service;
+
+  Future<void> fetchUnknownUris(
+    String isolateId,
```

Comment:
take a List of Strings as a param instead of using _unknownUris (which as I said in my previous comment should be tracked where `fetchUnknownUris` is being called.